packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1273,6 +1275,130 @@ describe('shallow', () => {
+          // 'formcontrol': HTMLFormControlsCollection,
+          form: HTMLFormElement,
+          frameset: HTMLFrameSetElement,
+          hr: HTMLHRElement,
+          head: HTMLHeadElement,
+          h1: HTMLHeadingElement,
+          h2: HTMLHeadingElement,
+          h3: HTMLHeadingElement,
+          h4: HTMLHeadingElement,
+          h5: HTMLHeadingElement,
+          h6: HTMLHeadingElement,
+          html: HTMLHtmlElement,
+          iframe: HTMLIFrameElement,
+          img: HTMLImageElement,
+          input: HTMLInputElement,
+          // isindex: HTMLIsIndexElement,
+          // keygen: HTMLKeygenElement,
+          li: HTMLLIElement,
+          label: HTMLLabelElement,
+          legend: HTMLLegendElement,
+          link: HTMLLinkElement,
+          map: HTMLMapElement,
+          media: HTMLMediaElement,
+          meta: HTMLMetaElement,
+          meter: HTMLMeterElement,
+          del: HTMLModElement,
+          ins: HTMLModElement,
+          ol: HTMLOListElement,
+          object: HTMLObjectElement,
+          optgroup: HTMLOptGroupElement,
+          option: HTMLOptionElement,
+          // 'options': HTMLOptionsCollection,
+          output: HTMLOutputElement,
+          p: HTMLParagraphElement,
+          param: HTMLParamElement,
+          // picture: HTMLPictureElement,
+          pre: HTMLPreElement,
+          progress: HTMLProgressElement,
+          blockquote: HTMLQuoteElement,
+          q: HTMLQuoteElement,
+          cite: HTMLQuoteElement,
+          script: HTMLScriptElement,
+          noscript: HTMLScriptElement,
+          select: HTMLSelectElement,
+          // shadow: HTMLShadowElement,
+          source: HTMLSourceElement,
+          span: HTMLSpanElement,
+          style: HTMLStyleElement,
+          caption: HTMLTableCaptionElement,
+          th: HTMLTableCellElement,
+          td: HTMLTableCellElement,
+          col: HTMLTableColElement,
+          colgroup: HTMLTableColElement,
+          table: HTMLTableElement,
+          tr: HTMLTableRowElement,
+          tfoot: HTMLTableSectionElement,
+          thead: HTMLTableSectionElement,
+          tbody: HTMLTableSectionElement,
+          template: HTMLTemplateElement,
+          textarea: HTMLTextAreaElement,
+          time: HTMLTimeElement,
+          title: HTMLTitleElement,
+          track: HTMLTrackElement,
+          ul: HTMLUListElement,
+          bgsound: HTMLUnknownElement,
+          blink: HTMLUnknownElement,
+          multicol: HTMLUnknownElement,
+          nextid: HTMLUnknownElement,
+          rb: HTMLUnknownElement,
+          spacer: HTMLUnknownElement,
+          video: HTMLVideoElement,
+        };
+
+        class Foo extends React.Component {
+          render() {
+            return (
+              <div id="container">
+                {Object.keys(map).map(Tag => <Tag key={Tag} />)}
+              </div>
+            );
+          }
+        }
+        const shallowWrapper = shallow(<Foo />);
```

Comment:
Convention is just to call this `wrapper`.
```suggestion
        const wrapper = shallow(<Foo />);
```